### DISCUSSION TITLE
Opção para redirecionar para a url da área do cliente ao inves do acesso de administrador.

### DISCUSSION THREAD

**Discussion by gildomatos** [RamonSilva20/mapos#2053] [General]
Gostaria de ajuda para redirecionar a url do mapos para a url da area do cliente. Atualmente ao acessar o link ele leva para a tela de login do administrador, mas no meu caso prefiro que ele inicie na tela do login de clientes.

Agradeço a atenção de quem puder me orientar. Qual arquivo devo alterar.

Obrigado.

**Comment by almron** [comment]
Esse foi um jeito não muito legal, elegante ou no olhar de um programador, um código limpo. Eu criei dois sites. Exemplo, meusite.com.br/mapos e outro meusite.com.br/cliente. Depois eu subi a última versão do Map-OS nos dois sites, porém apontando para o mesmo Banco de Dados. Em clientes, eu apontei para  https://meusite.com.br/mapos/index.php/mine.
Se alguém tiver um jeito melhor, eu agradeço.
